amgut/db/populate_test.sql

```diff
@@ -77,10 +77,10 @@ insert into project_barcode (project_id, barcode) values (1, '000010860'), (1, '
             '12/02/2013','foo','06:20 AM','','','N','N','N','','None','Y','N','N'),
         ('ded369ec-7d0f-4d8c-e040-8a80115d6742','d8592c74-7da2-2135-e040-8a80115d6401','000006616',19,'000006616.jpg','','Nasal mucus','08/12/2014',
             'MAKE SURE THIS IS BEING PASSED','12:05 PM','','','','','','','','','','');
-insert into ag_handout_kits (kit_id, password, barcode, verification_code, swabs_per_kit, print_results)
-    values('test_ha', '$2a$12$GfYI6sxlVGTamDJ3FEQzMuQ46eSV6YJ.DxUxiZvKJaAn1NJtTDtFS', '000000002', '5678', '3', 'n'),
-          ('test_ha', '$2a$12$GfYI6sxlVGTamDJ3FEQzMuQ46eSV6YJ.DxUxiZvKJaAn1NJtTDtFS', '000000003', '5678', '3', 'n'),
-          ('test_ha', '$2a$12$GfYI6sxlVGTamDJ3FEQzMuQ46eSV6YJ.DxUxiZvKJaAn1NJtTDtFS', '000000004', '5678', '3', 'n');
+insert into ag_handout_kits (kit_id, password, verification_code, swabs_per_kit, print_results)
+    values('test_ha', '$2a$12$GfYI6sxlVGTamDJ3FEQzMuQ46eSV6YJ.DxUxiZvKJaAn1NJtTDtFS', '5678', '3', 'n');
+insert into ag_handout_barcodes (kit_id, barcode)
+    values ('test_ha', '000000002'),('test_ha', '000000003'),('test_ha', '000000004');
```

Comment:
These changes really don't break any tests? That's surprising, and might indicate that our code test coverage should be improved.

For example, the `distinct` can be removed from `amgut.lib.data_access.ag_data_access.handoutCheck`. I suspect there are other changes that should be made, too, but will leave it to @squirrelo to investigate further. I gave it a quick look, and it appears that most of the functions that operate on the `ag_handout_kits` are untested, which makes me concerned that unforeseen errors will arise in the future.